packages/file_selector/file_selector_android/example/integration_test/file_selector_android_test.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('placeholder test', (WidgetTester tester) async {});
```

Comment:
Image picker has a test that would be similar, but apparently it doesn't work on FTL:

https://github.com/flutter/packages/blob/main/packages/image_picker/image_picker_android/example/android/app/src/androidTest/java/io/flutter/plugins/imagepickerexample/ImagePickerPickTest.java#LL31C1-L31C1

I assumed it would take a decent amount of time to find a test that worked that didn't have the same issue.